bgl/mcgregor.py

```diff
@@ -5,9 +5,7 @@
 
 # when computed on a leaf node this is equal to the number of edges mapped.
 def arcs_left(marcs):
-    # courtesy of jfass
-    return np.sum(np.any(marcs, 1))
-
+    return np.count_nonzero(marcs)  # slow
```

Comment:
Observation: Initially, this was counting the number of rows containing at least one nonzero element
https://github.com/proteneer/bgl/blob/ee8d958d8a0c6862d4471d1e98242b67ae23b165/mcgregor.py#L5-L11 , but this PR will change to counting the total number of nonzero elements.

Is this change okay?

(based on p. 30 of ref, I think the initial behavior was correct, but not sure if the change makes a difference in practice)

Reply:
Ahh -- makes sense!